docs/src/basic-tutorial.md

```diff
@@ -20,9 +20,9 @@ For more information, have a look at [Backends & Output](@ref).
 
 Ok, now that this is out of the way, let's get started!
 
-## Importing CairoMakie
+## First plot
 
-First, we import CairoMakie, which might take a little bit of time because there is a lot to precompile. Just sit tight!
+First, we import CairoMakie.
```

Comment:
I like the “sit tight”, it feels friendly

Reply:
The reason I removed this is that on 1.6, packages are precompiled by default, so this sentence doesn't really when loading the package.

Reply:
Ah true. What about

> First, we add and import CairoMakie, which might take a little bit of time because there is a lot to precompile. Just sit tight!

where the adding bit isn’t shown in code but presumably the user is often going to be adding it in order to use it. Also if we get the “auto suggest to add when you use” feature then importing could be triggering an add.